crates/sodium/src/hash.rs

```diff
@@ -14,6 +14,7 @@ pub const BYTES512: usize = rust_sodium_sys::crypto_hash_sha512_BYTES as usize;
 /// @param {SecBuf} input - the data to hash
 ///
 /// @param {SecBuf} output - Empty Buffer to be used as output
+#[deprecated(note = "use CryptoSystem")]
```

Comment:
What is purpose of this? To avoid ever breaking hc core? Are you going to audit hc core for deprecation warnings instead of build errors and make a separate PR in that repo?

Reply:
exactly, core uses a bunch of these legacy apis still, it's not a trivial amount of work to convert them over to crypto system..